addon/data/error-panel.js

```diff
@@ -23,6 +23,9 @@ self.port.on("showError", function (error) {
   if (error.stack) {
     console.error(error.stack);
   }
+  unsafeWindow.error = cloneInto(error, unsafeWindow);
+  unsafeWindow.captureException(unsafeWindow.error);
+
```

Comment:
Did you not go with the event approach?  I've found this approach fragile in weird ways (e.g., strings go across the boundary fine, but arrays act weird).  Though I'm unclear what cloneInto is?